packages/mdc-textfield/_mixins.scss

```diff
@@ -816,6 +837,18 @@
   }
 }
 
+@mixin mdc-text-field-placeholder-color_($color) {
+  .mdc-text-field__input::placeholder {
+    color: $color;
+  }
+
+  // Override the placeholder styles in IE with important rule to improve specificity.
+  // stylelint-disable-next-line selector-no-vendor-prefix
+  .mdc-text-field__input:-ms-input-placeholder {
+    color: $color !important;
```

Comment:
Almost same as above, except can we use `mdc-theme-prop(color, $color, $important: true)`?